content/applications/finance/accounting/payables/pay/multiple.rst

```diff
@@ -6,7 +6,7 @@ Odoo offers the possibility of grouping multiple bills' payments into one, facil
 reconciliation process.
 
 .. seealso::
-   - :doc:`../supplier_bills/manage`
+   - :doc:`../../../../inventory_and_mrp/purchase/manage_deals/manage`
```

Comment:
```suggestion
   - :doc:`/applications/inventory_and_mrp/purchase/manage_deals/manage`
```
You can start references from the `content` folder when it's a shorter and more readable path ;).